### PR TITLE
Update demo.md

### PR DESCRIPTION
**Pull request by mkostersitz**
Adding Windows PowerShell instructions Tab.

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by mkostersitz** [comment]
@song-jiang please review

**Comment by song-jiang** [comment]
Thanks @mkostersitz. Could you fix the CI issue below?

5:50:51 PM:       Generating...
5:51:02 PM:   Liquid Exception: Liquid syntax error (line 595): 'tabs' tag was never closed in getting-started/windows-calico/demo.md
5:51:02 PM:                     ------------------------------------------------
5:51:02 PM:       Jekyll 4.0.1   Please append `--trace` to the `build` command 
5:51:02 PM:                      for any additional information or backtrace. 
5:51:02 PM:                     ------------------------------------------------

**Comment by mkostersitz** [comment]
@song-jiang  fixed the missing  endtabs

File: getting-started/windows-calico/demo.md [hunk 1]
```diff
@@ -267,4 +270,332 @@ basic network policy to isolate pod to pod traffic. Finally, we can clean up all
...
+      value: This is a Calico for Windows demo.
+    imagePullPolicy: IfNotPresent
+  nodeSelector:
+    kubernetes.io/os: windows
+```
+
+### Apply the policy-demo-windows.yaml file to the Kubernetes cluster
+
+1. Open a powershell window.
+1. Use `kubectl` to apply the `policy-demo-windows.yaml` configuration
```
Comment (song-jiang) [inline getting-started/windows-calico/demo.md:393]: Suggest you add a point on how to add kubectl.exe to system path.

Comment (song-jiang) [reply getting-started/windows-calico/demo.md:393]: Use `kubectl` to apply the `policy-demo-windows.yaml` file created above.

File: getting-started/windows-calico/demo.md [hunk 2]
```diff
@@ -267,4 +270,332 @@ basic network policy to isolate pod to pod traffic. Finally, we can clean up all
...
+          app: busybox
+    ports:
+    - protocol: TCP
+      port: 80
+```
+
+### Apply the network-policy.yaml file
+
+1. Open a powershell window.
+1. Use `kubectl` to apply the network-policy.yaml file.
```
Comment (song-jiang) [inline getting-started/windows-calico/demo.md:555]: `network-policy.yaml`

**Review by song-jiang — COMMENTED** [review]
Just a few nits.

File: getting-started/windows-calico/demo.md [hunk 3]
```diff
@@ -267,4 +270,347 @@ basic network policy to isolate pod to pod traffic. Finally, we can clean up all
...
+                        charset=utf-8], [Date, Fri, 21 Aug 2020 22:45:46 GMT]}
+    Images            : {}
+    InputFields       : {}
+    Links             : {}
+    ParsedHtml        :
+    RawContentLength  : 49
+
+    ```
+
+### Conclusion
```
Comment (lmm) [inline getting-started/windows-calico/demo.md:531]:
```suggestion
```

Comment (lmm) [reply getting-started/windows-calico/demo.md:531]: We probably don't need this

**Review by lmm — COMMENTED** [review]
Looking good, added a bunch of formatting suggestions to fix things up

**Comment by lmm** [comment]
Btw @mkostersitz you can check out the preview here: https://deploy-preview-4473--calico-v3-18.netlify.app/getting-started/windows-calico/demo

**Comment by mkostersitz** [comment]
@lmm thanks for the review. I have accepted the suggestions. Makes sense.

**Review by lmm — APPROVED** [review]

**Comment by lmm** [comment]
Thanks a lot @mkostersitz !

**Pull request merged**